### PR TITLE
Add es-MX locale translations

### PR DESCRIPTION
**Pull request by VictorAvelar**
## Description

It adds support for spanish (Mexico) and sets it as default locale for spanish.

### CODE REVIEW

File: locale/es-MX.go [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	44 * time.Second:           func(_ time.Duration) string { return "hace unos segundos" },
+	89 * time.Second:           func(_ time.Duration) string { return "hace un minuto" },
+	44 * time.Minute:           func(d time.Duration) string { return fmt.Sprintf("hace %.0f minutos", math.Ceil(d.Minutes())) },
+	89 * time.Minute:           func(_ time.Duration) string { return "an hour ago" },
+	21 * time.Hour:             func(d time.Duration) string { return fmt.Sprintf("hace %.0f horas", math.Ceil(d.Hours())) },
+	35 * time.Hour:             func(_ time.Duration) string { return "a day ago" },
+	25 * (24 * time.Hour):      func(d time.Duration) string { return fmt.Sprintf("hace %.0f días", math.Ceil(d.Hours()/24.0)) },
+	45 * (24 * time.Hour):      func(d time.Duration) string { return "a month ago" },
+	10 * (24 * time.Hour) * 30: func(d time.Duration) string { return fmt.Sprintf("hace %.0f meses", math.Ceil(d.Hours()/(24.0*30))) },
+	17 * (24 * time.Hour) * 30: func(_ time.Duration) string { return "a year ago" },
```
Comment (patrickdevivo) [inline locale/es-MX.go:31]: I see this phrase (and some above) are still in English - I'm assuming they should also have Spanish versions? @VictorAvelar

**Comment by patrickdevivo** [comment]
great! thank you for contributing - left a comment about some phrases that still appear in English

File: locale/es-MX.go (hunk 1 above)
Comment (VictorAvelar) [reply locale/es-MX.go:31]: Yes you are correct, I missed those somehow. Will add them 😄. Thanks for the feedback

**Review by patrickdevivo — APPROVED** [review]
great - thank you for the contribution!

**Pull request merged**